lib/makeRequest.js

```diff
@@ -35,6 +35,7 @@ async function makeRequest(url, init = {}) {
 	}
 
 	throw createError(response.status, errors, {
+		responseBody: result,
```

Comment:
We access the error we still have to do `error.responseBody.errors[0].details` ... just curious would it be better if we spread the returned object or will that be an issue as we can't always guaranty what is returned from the API
```suggestion
		...result,
```

Reply:
A very interesting suggestion @kiyaGu as that would remove the extra `details` and/or `responseBody` layer 😄